packages/components/Button/src/ButtonTokens.ts

```diff
@@ -2,14 +2,18 @@ import { Theme } from '@fluentui-react-native/framework';
 import { globalTokens } from '@fluentui-react-native/theme-tokens';
 import { TokenSettings } from '@fluentui-react-native/use-styling';
 import { ButtonTokens } from './Button.types';
+import { Platform } from 'react-native';
 
 export const defaultButtonTokens: TokenSettings<ButtonTokens, Theme> = () =>
   ({
     block: {
       width: '100%',
     },
     medium: {
-      padding: globalTokens.spacing.sNudge - globalTokens.stroke.width.thin,
+       padding: Platform.select({
+        android: 6,
```

Comment:
nit: ideally spacing is defined by spacing tokens and not through hardcoded numbers

Reply:
I updated this to pick from another token instead of hardcoded value - the actual token work is still an open item.